src/microbe_stage/MembraneWaterRipple.cs

```diff
@@ -0,0 +1,666 @@
+﻿using System;
+using Godot;
+using Godot.Collections;
+
+/// <summary>
+///   Manages spawning and processing ripple effect for microbes
+/// </summary>
+public partial class MembraneWaterRipple : Node
+{
+    [Export]
+    public bool EnableEffect = true;
+
+    [Export]
+    public float RippleStrength = 0.8f;
+
+    [Export]
+    public float VerticalOffset = -0.05f;
+
+    /// <summary>
+    ///   Number of past positions to store for the movement history
```

Comment:
Does this need to match what is in the shader code? If so it would be pretty good to have warnings here not to change things without updating the shader as well.